packages/react-ui/src/widgets/WrapperWidgetUI.js

```diff
@@ -30,82 +30,101 @@ Actions props must have this format:
+  justifyContent: 'flex-start',
+  cursor: expandable ? 'pointer' : 'default',
+  '& .MuiButton-startIcon': {
+    marginTop: '3px',
+    marginRight: theme.spacing(1)
   },
-  header: ({ expanded }) => ({
-    display: 'flex',
-    flexDirection: 'row',
-    alignItems: 'flex-start',
-    justifyContent: 'space-between',
-    width: '100%',
-    ...(expanded ? { minHeight: theme.spacing(3) } : { height: theme.spacing(3) }),
-    padding: 0
+  '&:hover': {
+    background: 'none'
+  }
+}));
+
+const HideButton = styled(ExpandLess)(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  width: theme.spacing(3),
+  height: theme.spacing(3),
+  color: theme.palette.text.secondary
+}));
+
+const ShowButton = styled(ExpandMore)(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  width: theme.spacing(3),
+  height: theme.spacing(3),
+  color: theme.palette.text.secondary
+}));
+
+const Text = styled(Typography)(({ expanded = true }) => ({
+  wordBreak: 'break-word',
+  overflow: 'hidden',
+  ...(expanded && {
+    display: '-webkit-box',
+    WebkitLineClamp: 2,
+    WebkitBoxOrient: 'vertical'
   }),
-  optionsMenu: {
+  ...(!expanded && {
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis'
+  })
+}));
+
+const ActionsGrid = styled(Grid)(({ theme }) => ({
+  display: 'flex',
+  marginLeft: theme.spacing(1)
+}));
+
+const IconActionButton = styled(IconButton)(({ theme }) => ({
+  color: theme.palette.text.secondary,
+  margin: theme.spacing(-0.75, 0)
+}));
+
+const PaperMenu = styled(Menu)(({ theme }) => ({
+  '& .MuiDrawer-paper': {
     marginTop: theme.spacing(6),
     maxHeight: theme.spacing(21),
     minWidth: theme.spacing(16)
-  },
-  button: {
-    flex: 1,
-    padding: 0,
-    alignItems: 'flex-start',
-    justifyContent: 'flex-start',
-    cursor: (props) => (props.expandable ? 'pointer' : 'default'),
-
-    '& .MuiButton-startIcon': {
-      marginTop: '3px',
-      marginRight: theme.spacing(1)
-    },
-    '&:hover': {
-      background: 'none'
-    }
-  },
-  buttonText: ({ expanded }) => ({
-    wordBreak: 'break-word',
-    overflow: 'hidden',
-    ...(expanded && {
-      display: '-webkit-box',
-      WebkitLineClamp: 2,
-      WebkitBoxOrient: 'vertical'
-    }),
-    ...(!expanded && {
-      whiteSpace: 'nowrap',
-      textOverflow: 'ellipsis'
-    })
-  }),
-  actions: {
-    display: 'flex',
-    marginLeft: theme.spacing(1)
-  },
-  iconToggle: {
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
-    width: theme.spacing(3),
-    height: theme.spacing(3),
-    color: theme.palette.text.secondary
-  },
-  iconAction: {
-    color: theme.palette.text.secondary,
-    margin: theme.spacing(-0.75, 0)
-  },
-  content: {
-    paddingTop: theme.spacing(1.25)
   }
 }));
 
+const Content = styled(Box)(({ theme }) => ({
+  paddingTop: theme.spacing(1.25)
+}));
+
 function WrapperWidgetUI(props) {
```

Comment:
Probably you don't need to create a styled component for this and can add `pt={0.25}` helper directly to the `<Box />`